en/stripe-2.md

```diff
@@ -0,0 +1,261 @@
+2. On the endpoint list screen, click [Add new endpoint]. 
+3. In the Endpoint settings dialog, enter the following:
+
+| Field | Value |
+| :--- | :--- |
+| Category | Payments |
+| Model | Stripe, v1 |
+| Operation | checkout(\*) |
+| products_list  | Enter the price ID (Stripe API ID) of the product.<ul><li>Note: You can add multiple products to a single checkout.</li><li>Note: You can not change the price ID from the front end</li></ul> |
+| return_url     | Front-end URL for the payment success page. |
+| return_err_url | Front-end URL for the payment failure page. |
+
+[[info]] \*For the cancellation endpoint, select [cancel_order] for "Operation".
+
+4. When you are done, click [Add] to save the endpoint.
+
+
+### Verifying the billing process
+
+<!-- VIDEO: [![Image from Gyazo](https://t.gyazo.com/teams/diverta/405f30e1cfcef72cede37b86a45d1335.gif)](https://diverta.gyazo.com/405f30e1cfcef72cede37b86a45d1335) 
+[![Image from Gyazo](https://t.gyazo.com/teams/diverta/03fdaee9d13b521b773e7aed365d83e0.png)](https://diverta.gyazo.com/03fdaee9d13b521b773e7aed365d83e0)
+[![Image from Gyazo](https://t.gyazo.com/teams/diverta/cc0926ce21366db86a5a545014dfe375.png)](https://diverta.gyazo.com/cc0926ce21366db86a5a545014dfe375)
+[![Image from Gyazo](https://t.gyazo.com/teams/diverta/13a558163253ed7b19acf9805a2898a6.png)](https://diverta.gyazo.com/13a558163253ed7b19acf9805a2898a6)
+[![Image from Gyazo](https://t.gyazo.com/teams/diverta/e00d97f4fa571111e43c869924f2eba5.png)](https://diverta.gyazo.com/e00d97f4fa571111e43c869924f2eba5)
+[![Image from Gyazo](https://t.gyazo.com/teams/diverta/0029537b14a95543876fc455a60b3a91.png)](https://diverta.gyazo.com/0029537b14a95543876fc455a60b3a91)
+-->
+
+To check if your billing process is working as expected, verify the endpoint response from the Swagger UI and make a test payment.
+
+1. From the sidebar menu, select the API containing your checkout endpoint. On the endpoint list screen, click [Swagger UI].
+2. On the Swagger UI screen, select your checkout endpoint and click [Try it out].
+3. Scroll down and click [Execute].
+4. Copy the URL in the response body.
+
+[![Image from Gyazo](https://t.gyazo.com/teams/diverta/bd0480d5b7ba9a21dd703119387b99ac.jpg)](https://diverta.gyazo.com/bd0480d5b7ba9a21dd703119387b99ac)
+
+5. Access the URL in your browser. You should see the payment screen below. Enter your payment details and click [Subscribe].
+
+[[info]] You can use [test cards](https://stripe.com/docs/testing#international-cards) to simulate various scenarios.  
+
+6. If the payment was successful, you will be redirected to the payment success page you set in your endpoint.
+
+<!--
+Stripe-side verification steps needed
+-->
+
+Once completed, the member is automatically added to the member group(s) for the product(s). They will also receive a Stripe member ID and subscription ID attached to their Kuroco account. For more details, see [Second tutorial - Customer ID & Subscription ID](#link).
```

Comment:
Better to have screenshots for verification purposes.

Screenshot 1: Stripe's customer & subscription
- Reason: User should identify the payment's success by first checking if Stripe has processed it correctly. If nothing shows up in their Stripe, then it is a Stripe issue and they should look into Stripe rather than Kuroco.

Screenshot 2: Member's group info
- Reason: Once they confirmed that Stripe has processed and stored the information correctly on their end, then the user should check if Kuroco has also updated the information correctly for member by guiding the user to go to the Member list page, search for their member, and verify the group with the price_id that they made their payment for has been assigned to them.

Reply:
For screenshot 1 - Stripe, we can link to Stripe's documentation if therer are any that fits.